src/components/__tests__/Subthoughts.js

```diff
@@ -20,7 +20,7 @@ const whereContext = context => node => equalArrays(pathToContext(node.props().t
 
 // cannot figure out how to unmount and reset after each test so that we can use beforeEach
 beforeAll(async () => {
-  createTestApp()
+  await createTestApp()
```

Comment:
Good catch. Question is, why is `beforeAll` in this test file affecting tests in other files? Will you create an issue for this please? Thanks!

Reply:
I don't think it is affecting tests in other files. It was surprisingly working in this test without `await`.